ckanext/unaids/helpers.py

```diff
@@ -82,3 +83,15 @@ def get_user_obj(field=""):
     Returns an attribute of the user object, or returns the whole user object.
     """
     return getattr(g.userobj, field, g.userobj)
+
+
+def check_organization_update_access(organization_id):
```

Comment:
Registering an tuh function similar to registering a helper once you have implement IAuthFunctions.  We should probably register properly as an auth function as mentioned above.